### ISSUE TITLE
Mars Matrix

### ISSUE THREAD

**Issue by Brandonheat8** [Rinnegatamante/flycast-vita-Compatibility#80]
# Game summary
- Game name: Mars Matrix (US)
- Game version: v1.000

# flycast summary
- Version: Latest Release Build (All Modes)

# Issues
- Cutscenes run from 34-36FPS, Menus run at full 60FPS, Gameplay runs between 10-55FPS, Horrible lag and stuttering are constant with terrible glitches and rendering issues, the few glimpses of fullspeed are awesome but after about 2 minutes it hard crashes the Vita to the point of having to reboot it.

# Screenshots
![PS_App_20210108_060941](https://user-images.githubusercontent.com/76890866/104051739-db8f7e00-51a5-11eb-8668-347b796ce950.jpeg)
![PS_App_20210108_060943](https://user-images.githubusercontent.com/76890866/104051750-dfbb9b80-51a5-11eb-98fb-743b0851f586.jpeg)
![PS_App_20210108_060946](https://user-images.githubusercontent.com/76890866/104051759-e34f2280-51a5-11eb-86ad-613a4c03d69c.jpeg)
![PS_App_20210108_060948](https://user-images.githubusercontent.com/76890866/104051766-e5b17c80-51a5-11eb-8256-9a0485e86728.jpeg)
![PS_App_20210108_060959](https://user-images.githubusercontent.com/76890866/104051777-e9450380-51a5-11eb-9bcc-ca83cd7bd6b9.jpeg)
![PS_App_20210108_061002](https://user-images.githubusercontent.com/76890866/104051788-ed712100-51a5-11eb-8a93-abe5794d6563.jpeg)
![PS_App_20210108_063942](https://user-images.githubusercontent.com/76890866/104052125-77b98500-51a6-11eb-8e14-8afd4aefb06d.jpeg)

![PS_App_20210108_063945](https://user-images.githubusercontent.com/76890866/104051969-39bc6100-51a6-11eb-8117-5b37943c742b.jpeg)

![PS_App_20210108_060955](https://user-images.githubusercontent.com/76890866/104051986-404ad880-51a6-11eb-902b-42920a182387.jpeg)

# Recommended labels

- C - Ingame Low

**Comment by Rinnegatamante** [comment]
does it crash with smc builds too?

**Comment by Brandonheat8** [comment]
> does it crash with smc builds too?

Yes, without SMC it crashes normally, with SMC modes it hard gpu crashes the Vita.
